pkg/rdt/rdt.go

```diff
@@ -337,7 +337,7 @@ func (c *control) configureResctrl(conf config, force bool) error {
 	return nil
 }
 
-func (c *control) classesFromResctrlFs() (map[string]*ctrlGroup, error) {
+func (c *control) classesFromResctrlFs(all bool) (map[string]*ctrlGroup, error) {
```

Comment:
Hmm... I don't see the newly introduced `all bool` argument being used anywhere in the function.

Reply:
Argh, obviously my mental state on Friday evening was nearing coma and I had pushed some totally absurd patch. Where did I even find this one 🤔 Pushed the intended version, now